src/lib/config.ts

```diff
@@ -383,6 +383,10 @@ export class default_config {
         "<Space>": "hint.selectFocusedHint",
     }
 
+    browsermaps = {
+        "<C-,>": "escapehatch",
```

Comment:
Tempted to include a Ctrl-6 and Ctrl-^ (is that allowed?) bind here too, and remove it from the other modes.